mlrun/track/tracker.py

```diff
@@ -31,8 +31,9 @@ class Tracker(ABC):
     * Offline: Manually importing models and artifacts into an MLRun project using the `import_x` methods.
     """
 
+    @staticmethod
     @abstractmethod
-    def is_enabled(self) -> bool:
+    def is_enabled() -> bool:
```

Comment:
make sure to also change https://github.com/mlrun/mlrun/blob/10268fe0e814f6e9635eb085dbfad994569ed851/tests/track/test_tracker_manager.py#L28